infra/gravity/cluster_failover.go

```diff
@@ -0,0 +1,118 @@
+package gravity
+
+import (
+	"context"
+
+	"github.com/gravitational/robotest/lib/wait"
```

Comment:
Nit: We try to follow https://github.com/golang/go/wiki/CodeReviewComments#imports and thus organize all imports into 3 groups separated by an empty line: first stdlib, then imports from this project (`github.com/gravitational/robotest` in this case) and finally all other external imports.